sdk/core/azure-core-management/README.md

```diff
@@ -15,7 +15,7 @@ Azure Core Management library is a collection of classes common to the [Azure Re
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-management</artifactId>
-    <version>1.1.1</version>
+    <version>1.2.0</version>
```

Comment:
Are we releasing core-management library too?

Reply:
Yep!